tests/python/pytorch/graphbolt/impl/test_ondisk_dataset.py

```diff
@@ -1531,6 +1531,54 @@ def test_OnDiskDataset_preprocess_yaml_content_windows():
             )
 
 
+def test_OnDiskDataset_preprocess_force_reload(capsys):
```

Comment:
add dedicated testcase for `OnDiskDataset()` which is public API besides `preprocess_ondisk_dataset`(it's internal).